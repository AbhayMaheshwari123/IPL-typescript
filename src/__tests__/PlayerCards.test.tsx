import renderer from 'react-test-renderer'
import Team from '../Components/PlayerCards'

it('Teams', () => {
  const match={
    url: 'teams/chennai-super-kings',
  }
  const component = renderer.create(
    <Team
      match={{
        url: '',
      }}
      {...match}
    />
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
