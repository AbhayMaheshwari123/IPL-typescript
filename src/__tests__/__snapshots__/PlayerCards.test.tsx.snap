// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teams 1`] = `null`;
